i18n/en-US.properties

```diff
@@ -405,7 +405,9 @@ be.taskRejectErrorMessage = An error has occurred while rejecting this task. Ple
 # Button name to hide task assignee list
 be.taskShowLessAssignees = Show Less
 # Button name to expand task assignee list, additionalAssigneeCount is the number of additional task assignees that can be shown.
-be.taskShowMoreAssignees = Show {additionalAssigneeCount} More
+be.taskShowMoreAssignees = {additionalAssigneeCount, plural, one {Show # More} other {Show # More}}
```

Comment:
Is the `plural` specification needed? It looks like the message variants for `one` and `other` are the same.

Reply:
in English, yes, but @ehoogerbeets pointed out it is different in other languages